plugin/src/main/java/io/jenkins/plugins/casc/core/JenkinsConfigurator.java

```diff
@@ -56,11 +55,22 @@ protected Jenkins instance(Mapping mapping, ConfigurationContext context) {
                 .setter( noop() ));
 
         // Override "nodes" getter so we don't export Nodes registered by Cloud plugins
-        Attribute.<Jenkins,List<Node>>get(attributes, "nodes").ifPresent(a ->
-            a.getter(j -> j.getNodes().stream()
-                    .filter(node -> node.getDescriptor().isInstantiable())
-                    .collect(Collectors.toList())
-            )
+        Attribute.<Jenkins, List<Node>>get(attributes, "nodes").ifPresent(attribute ->
+                attribute
+                        .getter(jenkins -> jenkins.getNodes().stream()
+                                .filter(node -> !isCloudNode(node))
```

Comment:
Just a suggestion :sweat_smile: 
For simplicity, you could have a method `isNotCloudNode` and use a method reference.

Reply:
Done before you mentioned it 👍

Reply:
not done?
Casz meant something like:
```suggestion
                                .filter(this::isNotCloudNode)
```

Reply:
Once I need `isCloudNode` and once `isNotCloudNode`

Reply:
Yes sorry for not using suggestions

Reply:
`isNotCloudNode` will `return !isCloudNode()`